common/display/displayplanemanager.cpp

```diff
@@ -467,11 +455,9 @@ void DisplayPlaneManager::ValidateCursorLayer(
     overlay_begin = overlay_planes_.begin() + composition.size();
   }
 
-#ifdef DISABLE_CURSOR_PLANE
   overlay_end = overlay_planes_.end() - 1;
   if (total_size == 1)
     overlay_begin = overlay_planes_.begin() + composition.size();
-#endif
```

Comment:
Why lines in DISABLE_CURSOR_PLANE needs to remain

Reply:
Yes, these lines should be removed. I will update soon